demosplan/DemosPlanUserBundle/Repository/AddressRepository.php

```diff
@@ -155,7 +155,7 @@ public function generateObjectValues($entity, array $data)
             $entity->setUrl($data['url']);
         }
 
-        //## Boolean
```

Comment:
wtf?! Is that supposed to be a comment header?! Can we maybe teach the csfixer to turn that simply into `//`?